### DISCUSSION TITLE
Want to add php artisan make:crud ClassName command as new feature

### DISCUSSION THREAD

**Discussion by mnszaidi** [laravel/framework#39781] [Ideas]
Hi i am using this custom command in my  projects of Laravel, i want that should be included in Next version of laravel it will make Laravel development very Fast and oriented, This command will Create Model, Migrations, Controller, Routes for the Methods defined in Controller, Views, Observer, Notification and Export File as well, We can minify it to Model, Migrations, Controller, Routes and Views for simple version. I do have Working Command and Stubs, i can contribute to adding this Feature

**Comment by crynobone** [comment]
Hey there,

We're closing this issue because it's inactive, already solved, old, or not relevant anymore. Feel free to open up a new issue if you're still experiencing this problem.
